website/source/docs/providers/aws/d/route_table.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "aws"
+page_title: "AWS: aws_route_table"
+sidebar_current: "docs-aws-datasource-route-table"
+description: |-
+    Provides details about a specific Route Table
+---
+
+# aws\_route\_table
+
+`aws_route_table` provides details about a specific Route Table.
+
+This resource can prove useful when a module accepts a Subnet id as
+an input variable and needs to, for example, add a route in 
+the Route Table.
+
+## Example Usage
+
+The following example shows how one might accept a Route Table id as a variable
+and use this data source to obtain the data necessary to create a route.
+
+```
+variable "subnet_id" {}
+
+data "aws_route_table" "selected" {
+  subnet_id = "${var.subnet_id}"
+}
+
+resource "aws_route" "route" {
+  route_table_id = "${data.aws_route_table.selected.id}"
+  destination_cidr_block = "10.0.1.0/22"
+  vpc_peering_connection_id = "pcx-45ff3dc1"
+}
+```
+
+## Argument Reference
+
+The arguments of this data source act as filters for querying the available
+Route Table in the current region. The given filters must match exactly one
+Route Table whose data will be exported as attributes.
+
+
+* `filter` - (Optional) Custom filter block as described below.
+
+* `id` - (Optional) The id of the specific Route Table to retrieve.
```

Comment:
Was this parameter implemented? I can't see any reference to it in `data_source_aws_route_table.go` and get this when I try to use it,
```
Errors:
  * data.aws_route_table.selected: : invalid or unknown key: id
```